terraform/modules/github_actions_dev_terraform_roles/policies.tf

```diff
@@ -0,0 +1,84 @@
+data "aws_region" "current" {}
+
+data "aws_caller_identity" "current" {}
+
+data "aws_iam_openid_connect_provider" "github" {
+  arn = "arn:aws:iam::${data.aws_caller_identity.current.account_id}:oidc-provider/token.actions.githubusercontent.com"
+}
```

Comment:
Actually this should use a variable passed in from main.tf, not a data block (that's why the plan is failing)

Reply:
Ah - I've changed this now but how do I set this in the staging main.tf file where the github_actions_openid_connect_provider is not being used?

Reply:
We don't need this module in staging. If we did there's already a data block in staging/main.tf. It's OK that the staging plan is currently failing, it should start passing once we do an apply on test.

If you're happy shall I rebase, squash, and merge?